browser/src/Editor/NeovimEditor/NeovimEditor.tsx

```diff
@@ -254,6 +254,35 @@ export class NeovimEditor extends Editor implements IEditor {
             notification.show()
         })
 
+        const initVimPath = this._neovimInstance.getInitVimPath()
+        const exists = Boolean(initVimPath)
+        const initVimInUse = this._configuration.getValue("oni.loadInitVim")
+        if (exists && typeof initVimInUse === "undefined") {
```

Comment:
Ah, I think I see what's happening - we're using `undefined` to track the state where we haven't done this check yet, and `false` to explicitly say that we've opted out. I guess that's why the config change below is necessary. That seems a bit confusing to me though - using `undefined` as a sentinel is a bit scary.

Could we perhaps instead have an explicit value, tracking if we've done this check? Like, in the case where the user presses `yes` or `no`, we  set an explicit value, like `this._configuration.setValue("_internal.hasCheckedInitVim", true, true /*persist*/)`. Then, we use that to decide whether or not we should do this check - instead of the implicit `undefined`/`false`/ differentation.